src/deck-options.md

```diff
@@ -4,260 +4,221 @@
-v1 schedulers, the counts are affected by parents/selected decks in the same way
-as new cards.
+When studying a deck that has subdecks inside it, the review limit behaves similar to the new card limit.
 
-In the v2 scheduler, the limit is taken solely from the deck you select - any
-limits on its parents or child decks are ignored.
+Anki includes any learning cards that has [crossed the day boundary](https://docs.ankiweb.net/preferences.html#scheduler) (interday learning cards) in the review count,
+so those learning cards will be subject to the review limit.
 
-The v3 scheduler includes learning cards with a 1+ day delay in the review count,
-so those learning cards will be subject to the daily limit.
+#### Per-Deck Daily Limits
 
-### New Cards Ignore Review Limit
+It is possible to use the same preset for different decks, with customized limits for each one of them. This eliminates the need to create cloned presets just for that purpose, and makes it easier to set custom limits on subdecks when you have many of them.
 
-If using [the v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits),
-please keep in mind that the new count is capped by the review count by default. If your
-review limit is set to 200, and you have 190 reviews waiting, a maximum of 10
-new cards will be introduced. If your review limit has been reached, no new
-cards will be shown. If you have a backlog of reviews and still want to
-introduce new cards, you can do so by suspending the reviews, or increasing your
-review limit. That said, it is recommended you hold off on new cards until you
-catch up instead, as introducing more new cards when you're behind will only
-make the backlog worse.
+Anki provides three options for daily limits:
 
-From Anki 2.1.61 this feature is optional, and can be deactivated globally from the
-deck options screen.
+- **Preset**: This limit applies to all the decks using a preset.
+- **This deck**: This limit is specific to a particular deck.
+- **Today only**: This allows changing the limit of a deck for a day.
 
-### Per-Deck Daily Limits
+### New Cards Ignore Review Limit
 
-From version 2.1.55 it is possible to use the same preset for different decks / subdecks, with customized
-limits for each one of them. This eliminates the need to create cloned presets just for that
-purpose, and makes it easier to set custom limits on sub-decks when you have many nested decks.
+By default, the review limit also applies to new cards, and no new cards will be shown when the review limit has been reached. If this option is enabled, new cards will be shown regardless of the review limit. 
 
-The options are:
+If your review limit is set to 200, and you have 190 review cards waiting, a maximum of 10
+new cards will be introduced. If your review limit has been reached, then Anki won't show you any new cards. This is useful if you have a backlog of review cards.
```

Comment:
```suggestion

```

The clarified language above expresses this well, so I don't think we need an example anymore. 

It also disrupts the flow of the subsection, since it first talks about the default, then about the setting, and then in more detail about the default.